src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -193,9 +200,67 @@ export const SiteDashboard = (): JSX.Element => {
           {/* Right column */}
           <Box w="40%">
             <VStack spacing="1.25rem">
+              {/* Site launch display card */}
+              {isUserUsingSiteLaunchFeature(siteName) &&
```

Comment:
i think this is fine as-is, but we might consider making a utility component such as `FeatureFlag`, which exposes a `showFeature` prop that renders the child component iff `showFeature` is `true`. 

this is still isolated to only this portion of the codebase so it's fine for now (assuming that we're moving to launchdarkly soon) but otherwise it'd be more important

Reply:
yup agreed!